frontend/packages/employee-frontend/src/components/person-profile/PersonApplications.tsx

```diff
@@ -123,3 +123,19 @@ const PersonApplications = React.memo(function PersonApplications({
 })
 
 export default PersonApplications
+
+export function inferApplicationType(application: ApplicationSummary) {
+  const baseType = application.type.toUpperCase()
+  if (baseType !== 'PRESCHOOL') return baseType
+  else if (
+    application.connectedDaycare &&
+    !application.preparatoryEducation
+  ) {
+    return 'PRESCHOOL_WITH_DAYCARE'
+  } else if (
+    application.connectedDaycare &&
+    application.preparatoryEducation
+  ) {
+    return 'PREPARATORY_WITH_DAYCARE'
+  } else return 'PRESCHOOL'
```

Comment:
and maybe something like:
```suggestion
  if (!application.connectedDaycare) return 'PRESCHOOL'
  if (application.preparatoryEducation) {
    return 'PREPARATORY_WITH_DAYCARE'
  } else {
    return 'PRESCHOOL_WITH_DAYCARE'
  }
```

but that's just a preference, too.